doc/source/packaging/diag/python_library_diag.rst

```diff
@@ -0,0 +1,66 @@
+.. _python pkg lib diag:
+.. graphviz::
+    :caption: A Python library is a collection of packages.
+    :alt: A Python library is a collection of packages.
```

Comment:
None of the other captions have a period concluding them--but this one is a sentence so I wasn't sure whether to delete the period or keep it.

Reply:
I think figure captions need the period at the end of their caption. Will modify the rest accordingly.